identity/handler.go

```diff
@@ -175,11 +175,12 @@ type listIdentitiesParameters struct {
 //	Responses:
 //	  200: listIdentities
 //	  default: errorGeneric
-func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+func (h *Handler) list(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 	var (
 		err    error
 		params = ListIdentityParameters{
 			Expand:                       ExpandDefault,
+			IdsFilter:                    r.URL.Query()["ids"],
```

Comment:
This new parameter also needs to be mentioned in `listIdentitiesParameters`, so that we have a proper API documentation. Unfortunately we don't use that struct to actually parse the request query, but it still shows up in the OpenAPI spec.